app/services/activity/import.rb

```diff
@@ -30,6 +30,34 @@ def is_oda_by_type(type:)
           non_ispf: nil
         }[type]
       end
+
+      def unexpected_non_oda_attribute_errors(activity, converted_attributes)
+        level = {
+          "programme" => :level_b,
+          "project" => :level_c_d,
+          "third_party_project" => :level_c_d
+        }[activity.level]
```

Comment:
Maybe we should just pass in the level like we do with `filtered_csv_column_headings`